### ISSUE TITLE
Use high contrast icons for conversation info

### ISSUE THREAD

**Issue by marcoambrosini** [nextcloud/spreed#13175]
* Much better readability of the information. The contrast of the star is very poor and the icon size of the link too small.
* Avatar image more visible

### Now
<img width="578" alt="Screenshot 2024-08-29 at 11 54 43" src="https://github.com/user-attachments/assets/5ac7f03d-8b47-4f8c-bda8-bd62c6d74b6c">

### Proposed change 
<img width="575" alt="Screenshot 2024-08-29 at 11 54 48" src="https://github.com/user-attachments/assets/438548dd-dc9f-49f9-9ef7-507b2688da3f">

**Comment by marcoambrosini** [comment]
Can also be `maxcontrast` if you think that's too present 

<img width="813" alt="Screenshot 2024-08-29 at 11 59 05" src="https://github.com/user-attachments/assets/e163689a-25cd-4c5f-82e8-84a9c0d64d2e">

Still much better imo

**Comment by jancborchardt** [comment]
Nice, looks very very clean actually. :) I like it!

**Comment by Antreesy** [comment]
`max-contrast` fits better here, in my opinion. Shall we schedule it for 31?

**Comment by ShGKme** [comment]
I'd vote for the old design. It allows to add more text information on the the date/time of the last message while keeping icons on the conversation icon.

**Comment by jancborchardt** [comment]
> We also have the "red Camera icon" when a call is on going and I can't see that being inlined, but maybe it could be a "red (blinking?) border" on the avatar.

@nickvergessen yeah, that sounds good, like a "live" pulsing effect as a nice animation. @marcoambrosini?

> Also I think the yellow from the favorite should stay somehow, as there is 🌐 for federation and ⛓️ for public there are otherwise quite many icons that try to raise the same importance as my favorites?

@nickvergessen agreed as well, also makes it stand out a bit more like favorites should. Needs to be an accessible yellow like in Files @marcoambrosini.

**Comment by jancborchardt** [comment]
> What about using a green pulsing effect instead of a red one?
> Since the user will see a green join call button when entering that conversation, I think green will fit better.
> We might use a red pulse only when the call is being recorded? (that might be overthinking and it might not be self-explainatory enough)

A red dot, or red pulsing is always color code to communicate that something is "Live", which a call is. I don’t think we should do any custom color coding like green here. If the conversation is recorded, people need to accept it in the modal, and there it's properly explained.

**Comment by nickvergessen** [comment]
We can make it much more yellow with AA and use the brown for AAA as discussed somewhere another day.

**Comment by marcoambrosini** [comment]
The color above is barely 3:1 (AA) with our rather bright background on this navigation. I don't see how we can meet the requirements with a brighter color. Could you or someone that was in that discussion provide source or docs about this?

**Comment by jancborchardt** [comment]
@marcoambrosini accessible yellow aka dark orange / gold is still better than grey among other icons for a star that’s supposed to stand out. :)

**Comment by nickvergessen** [comment]
Things to keep in mind.

There are various requests to better outline some more states like other messengers do:
- Muted conversations should show a 🔕  (WhatsApp, Telegram, …)
- Archived conversations should show a 📦 during search (WhatsApp, Telegram, …)
- Expiring should show ⏱ (WhatsApp, Telegram, …)
- Federation should show 🌍

**Comment by DorraJaouad** [comment]
![Image](https://github.com/user-attachments/assets/f508651d-9ee6-4482-8b00-0be37d93f268)

**Comment by Antreesy** [comment]
From team week discussion:

- [x] Remove favorite and use tag layout - #18927 
- [ ] Add time top-right (H:i 13:07 for 18h, D Mon for 5 days, d. M 15. Jun / M/d Jun/15 for 10 months, d.M.Y 15.06.26 afterwards) - #18987
- [ ] Remove public chain ⇒ show as toggle in right sidebar and clicking opens the conversation settings
- [ ] Remove federation icon ⇒ Show in right sidebar with info about server
- [ ] Remove call icon and replace by red border around avatar (joas/andy style)
- [ ] Other icons discussed in the PR are not needed
- [ ] Add “muted” icon for disabled notifications at the end of name (telegram style)
